src/properties/devServer/index.test.js

```diff
@@ -4,67 +4,67 @@ import { urlPart } from '../../types'
 
 const validModuleConfigs = [
   // #0
```

Comment:
Just a random comment. Any idea why we are maintaining these numbers? It's a little painful to add new tests to the middle.